### DISCUSSION TITLE
Is there a way to order records by dragging?

### DISCUSSION THREAD

**Discussion by NaungYeHtet** [filamentphp/filament#10347] [Help] [answered]
I have a table faqs which has columns title, detail and order.

Currently using form to order the records.

Is there a way to sort order with more efficient in user experience?

**Comment by MedanWeb** [comment, marked as answer]
Using drag and drop [https://filamentphp.com/docs/3.x/tables/advanced#reordering-records](url)

**Comment by parusel** [comment]
Getting 404 response when following the link. Is there an update on this topic? :-)

**Reply by dissto** [reply]
Copy the [link](https://filamentphp.com/docs/3.x/tables/advanced#reordering-records) manually and paste it in the url bar 😊
